In [3]:
import pandas as pd
import numpy as np

In [5]:
data = pd.read_csv("iris.csv")
data

sepal.length  sepal.width  petal.length  petal.width    variety
0             5.1          3.5           1.4          0.2     Setosa
1             4.9          3.0           1.4          0.2     Setosa
2             4.7          3.2           1.3          0.2     Setosa
3             4.6          3.1           1.5          0.2     Setosa
4             5.0          3.6           1.4          0.2     Setosa
..            ...          ...           ...          ...        ...
145           6.7          3.0           5.2          2.3  Virginica
146           6.3          2.5           5.0          1.9  Virginica
147           6.5          3.0           5.2          2.0  Virginica
148           6.2          3.4           5.4          2.3  Virginica
149           5.9          3.0           5.1          1.8  Virginica

[150 rows x 5 columns]

In [8]:
iris[iris.duplicated(),]

NameError: name 'iris' is not defined

#### Observations
- There are 150 rows and 5 column
- There are no Null value
- There are no duplicated values
- The x-column are sepal.length, sepal.width, petal.length
- All the x-columns are continuous
- The y-column is "variety"which is categorical
- The are three flower categories